.prettierrc

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Adding prettier is out of scope for this PR.
I guess adding eslint is a good idea

Reply:
it was too many files , so it is hard to write without formating
I can remove it

Reply:
removed

Reply:
Please also revert prettier changes - i plan to add eslint soon + validate it on ci

Reply:
reverted prettier of the most of the unchanged files